streamlit_drawable_canvas/frontend/src/lib/circle.ts

```diff
@@ -55,7 +56,9 @@ class CircleTool extends FabricTool {
       evented: false,
       radius: this._minRadius,
     })
-    canvas.add(this.currentCircle)
+    if (_clicked === 0) {
+      canvas.add(this.currentCircle)
+    }
```

Comment:
Because we have enabled right-click for all the drawing tools, I think it's better to prevent right-click from drawing anything or it might confuse the users.

The if-statement here limit the drawing function to left-mouse-button only. The other `*.ts` files have similar lines for the same reason.